src/coreclr/pal/src/libunwind/libunwind-version.txt

```diff
@@ -5,3 +5,6 @@ Replace CMakeLists.txt, src/CMakeLists.txt, configure.cmake with .NET custom ver
 Keep .NET oop directory
 Reapply changes from https://github.com/dotnet/runtime/commit/1b5719c2e3dde393531eaeb5b5cde05dabeef5b8
 Apply https://github.com/libunwind/libunwind/pull/317
+
+For LoongArch64:
+Apply https://github.com/libunwind/libunwind/pull/316
```

Comment:
Also should be added `Apply https://github.com/libunwind/libunwind/pull/322`

Reply:
At least if I undertand things correctly.

Reply:
> Also should be added `Apply https://github.com/libunwind/libunwind/pull/322`

I don't see `atomic_bool` changes in this PR.

Reply:
I pushed the `atomic_bool` changes to the upstream.
If need, I will update it within this PR.

Reply:
merged the `https://github.com/libunwind/libunwind/pull/322` by https://github.com/dotnet/runtime/pull/62979/commits/db234e52bef1fe49cf107535a8fed33bf771d82d